3.analyze_data/UMAP_analysis/nbconverted/3.vis_mito_jump.r

```diff
@@ -0,0 +1,202 @@
+suppressPackageStartupMessages(library(dplyr))
```

Comment:
you have become an R expert!!! 🥇

Reply:
Thank you! 🎉 I will only partially accept this award until I figure out patchwork and THEN I will truly feel like an expert 😆